mcs/tools/corcompare/Util.cs

```diff
@@ -7,18 +7,29 @@ namespace Mono.ApiTools {
 
 	class TypeHelper {
 
+		public TypeHelper (bool ignoreResolutionErrors)
+		{
+			IgnoreResolutionErrors = ignoreResolutionErrors;
+		}
+
+		public bool IgnoreResolutionErrors { get; }
+
 		public AssemblyResolver Resolver { get; } = new AssemblyResolver();
 
 		internal bool IsPublic (TypeReference typeref)
 		{
 			if (typeref == null)
 				throw new ArgumentNullException ("typeref");
 
-			TypeDefinition td = typeref.Resolve ();
-			if (td == null)
-				return false;
+			try {
+				var td = typeref.Resolve ();
+				if (td == null)
+					return false;
 
-			return td.IsPublic;
+				return td.IsPublic;
+			} catch (AssemblyResolutionException) when (IgnoreResolutionErrors) {
+				return true;
```

Comment:
If we can't find this type, just assume it is public. If we really cared, then we should have added the dependency to the resolver. If we want this to fail, then don't set the flag to true.